spec/v2/index.bs

```diff
@@ -2169,6 +2183,26 @@ path: LICENSE.md
 
 # Appendix B: Changelog # {#changelog}
 
+
+## Version 2.0.1-20230301 (Mar 1, 2023) ## {#changelog-2.0.1-20230301}
+
+This version fixes 2 discrepancies between the Pathfinder Framework 
+Version 2 and the data model in this specification.
+
+1. property <{CarbonFootprint/boundaryProcessesDescription}>: was defined as
+    optional is now mandatory in accordance with Pathfinder Framework Version 2
```

Comment:
to clarify, I would write: "was incorrectly defined as optional in v2.0.0, and this typo is now corrected such that the property is correctly marked as mandatory in accordance with the Pathfinder Framework Version 2"